### DISCUSSION TITLE
120 VAC Dosing Pump?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#969] [REM]
I need to replace one of my dosing pumps, maybe both.

They are 24V pumps from an Intellichem setup, but now driven from a relay card on an rpi.  

The Econ series from Stenner will fit on the 15 gallon tanks I have, but they are 120 VAC.  

Has anyone used those with an SSR or maybe a Z-wave or Zigbee AC outlet?

Alternately, anyone have a suggestion for a  24V dosing pump?  Or, a 120V AC pump that has an SSR that could be controlled from the 24V output sfrom the REM box?

**Comment by davthomaspilot** [comment]
Turns out the Econ FX series come in both 24v and 110 VAC versions.  

Found the 24 volt version on EBAY and ordered 2.

https://www.ebay.com/itm/155109483980

The rep at Stenner says the check valves are needed for these pumps and should be removed.  One less thing to replace each year I guess

**Comment by TopJim** [comment]
Little late for you now, but I had originally ordered the 120 v Econ FX, only to find that I could not use them as I had wanted to and that I needed the 24 VDC. I removed the transformers inside the Stenners, added a switch, snubber, and wired 24 VDC from my relay. Power to the relay was provided by another transformer. For check valve on the acid injection I went with the Stenner Ball Check Valve - 1/4" Tantalum Spring (FKM) which should last a good long while.
Switch https://www.amazon.com/gp/product/B0CF4L549N/ref=ppx_yo_dt_b_asin_title_o07_s02?ie=UTF8&psc=1
Snubber https://www.amazon.com/gp/product/B0BG2BGJYZ/ref=ppx_yo_dt_b_asin_title_o04_s00?ie=UTF8&psc=1
Pwr supply https://www.amazon.com/gp/product/B0B1DCR6WP/ref=ppx_yo_dt_b_asin_title_o04_s00?ie=UTF8&psc=1

**Reply by davthomaspilot** [reply]
Thanks for the update--mine should be here this afternoon.

I already have the 24V supply, since that's what I was running the Intellichem dosing pumps with.  I think the Intellichem are the same pump as the Econ Series (24V version), just in a different enclosure designed by Pentair to mount on their tanks.

The Stenner rep said I didn't need a check valve and that I shouldn't use one.  He said it would just clog eventually and have to be replaced.  (I can confirm that I have to change them yearly))
